lib/node_modules/@stdlib/math/base/assert/is-probability/examples/c/example.c

```diff
@@ -0,0 +1,34 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/assert/is_probability.h"
+#include <math.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+int main() {
+    double x;
+    bool v;
+    int i;
+    
+    for ( i = 0; i < 100; i++ ) {
+        x = ( ( ( double ) rand() / ( double ) RAND_MAX ) * 2.0 ) - 1.0;
```

Comment:
```suggestion
        x = ( ( (double)rand() / (double)RAND_MAX ) * 2.0 ) - 1.0;
```

For casts, by convention, we don't surround the type with spaces.